frontend/src/metabase/visualizations/visualizations/Text/Text.jsx

```diff
@@ -202,3 +204,13 @@ export default class Text extends Component {
     );
   }
 }
+
+function getMarkdownLinkTarget(link) {
```

Comment:
![image](https://user-images.githubusercontent.com/1937582/206421265-1a01f2b6-6f1d-461d-a365-0e04d46ac863.png)

Thank you! I'd been searching for it for a while but gave up.

Reply:
@ranquild I ended up removing the function entirely and not even using `getUrlTarget` since the only way we could configure `rel` is to use [rehype-external-links](https://github.com/rehypejs/rehype-external-links). Which handles detecting external links for us already.